catalog/core/catalog-core-api-impl/pom.xml

```diff
@@ -119,6 +157,19 @@
                     <skip>false</skip>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>1.4</version>
```

Comment:
I know there's a couple of other places that use gmavenplus - should we move this plugin to pluginManagement?